docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -0,0 +1,142 @@
+
+![img](/images/yb-platform/kubernetes-config1.png)
+
+Note that the cloud provider example used in this document has a cluster-level admin access.
+
+## Create a universe
+
+If no universes have been created yet, the Yugabyte Platform Dashboard looks similar to the following:
+
+![img](/images/yb-platform/kubernetes-config2.png)
+
+To start, click **Create Universe** and complete the first two fields of the **Cloud Configuration** section shown in the following illustration:
+
+![img](/images/yb-platform/kubernetes-config3.png)
+
+- In the **Name** field, enter the name for the YugabyteDB universe (or example, yb-k8s).
+
+- Use the **Provider** field to select the appropriate Kubernetes cloud (or example, K8s Provider). Notice that additional fields are now displayed, as shown in the following illustration:
+
+   ![img](/images/yb-platform/kubernetes-config4.png)
+
+Complete the rest of the **Cloud Configuration** section as follows:
+
+- Use the **Region** field to select the region. This enables the **Availability Zones** option that allows you to see zones belonging to that region. 
+
+- Provide the value in the **Pods** field. This value should be equal to or greater than the replication factor. The default value is 3. When this value is supplied, the pods are automatically placed across all the availability zones to guarantee the maximum availability.
+
+- In the **Replication Factor** field, define the replication factor. The default is 3. 
+
+  ![img](/images/yb-platform/kubernetes-config5.png)
+
+Complete the **Instance Configuration** section as follows:
+
+- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer pods. The default is Small (4 Cores & 7.5GB RAM). You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
+
+- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1*100GB.
+
+- Use the **Enable YSQL** field to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
+
+- Use the **Enable YEDIS** field to enable the YEDIS API endpoint to run REDIS-compatible workloads. This setting is disabled by default.
+
+- Use the **Enable Node-to-Node TLS** field to enable encryption-in-transit for communication between the database servers. This setting is disabled by default.
```

Comment:
We should mention somewhere the Platform version which we've used while preparing the doc. Otherwise, we need to modify this doc if we're referring it to the latest platform version. This flag default enabled, v2.7.1.1 onwards. 

![image](https://user-images.githubusercontent.com/13635843/123227330-1b4ae780-d4f2-11eb-83d6-449ac19f4465.png)

Reply:
Added a note about the version and replaced the image